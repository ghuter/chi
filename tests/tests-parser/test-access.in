x :: pts.x;
y :: pts.y;
sum :: pts.x + pts.y;
