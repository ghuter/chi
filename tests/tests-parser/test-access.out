const SDECL(x, EACCESS(EMEM(pts).x) : ?) : ?
const SDECL(y, EACCESS(EMEM(pts).y) : ?) : ?
const SDECL(sum, EBINOP(ADD, EACCESS(EMEM(pts).x) : ?, EACCESS(EMEM(pts).y) : ?) : ?) : ?
